static/docs/command-reference/get.md

```diff
@@ -66,12 +67,12 @@ created in the current working directory, with its original file name.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Examples
+## Example: Retrieve a model from a DVC remote
 
 > Note that `dvc get` can be used from anywhere in the file system, as long as
 > DVC is [installed](/doc/install).
 
-We can use `dvc get` to download the resulting model file from our
+We can use `dvc get` to obtain the resulting model file from our
 [get started example repo](https://github.com/iterative/example-get-started), a
```

Comment:
In this example also we're talking about a remote URL, so the term "download" was correct.

Reply:
Sorry, I know it may sound like I'm contradicting myself in https://github.com/iterative/dvc.org/pull/821/#discussion_r351033343, but I didn't mean to just replace all instances of the word 🙂